File: input/InputPrint.cpp
#include <iostream>

using namespace std;

int main()
{
	cout << "Please input size of array: " << endl;
	int size;
	cin >> size;
	cout << "size is: " << size << endl;
	int a[size];
	
	cout << "Please input elements of array: " << endl;
	for (int i = 0; i < size; i++)
	{
		cin >> a[i];
	}
	
	for (int i = 0; i < size; i++)
	{
		cout << a[i] << endl;
	}
	return 0;
}

File: input/InputPrint.java

import java.util.Scanner;

public class InputPrint
{
	
	public static void main(String[] args)
	{
		System.out.println("Please input size of array: ");
		Scanner scan = new Scanner(System.in);
		int size = scan.nextInt();
		int a[] = new int[size];
		System.out.println("Please input elements of array: ");
		for (int i = 0; i < size; i++)
		{
			a[i] = scan.nextInt();
		}
		scan.close();
		for (int i = 0; i <  a.length; i++)
		{
			System.out.println(a[i]);
		}
	}
}